lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -33,6 +33,7 @@ sub verify_incident_repos ($url_handler, $incident_repos) {
 }
 
 sub detect_maintenance_update ($jobid, $url_handler, $settings) {
+    return undef if $settings->{PUBLIC_CLOUD_SKIP_MU};
```

Comment:
Could we do it more generic? I mean, instead of specific public cloud var, just generic var `SKIP_MU|SKIP_UPDATES|SKIP_MAINTENANCE_UPDATES` ?

Reply:
Changed :)